build_bundle_builder_image.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+source ./_common.sh
+
+function build_docker_image {
+	#Later this might need a revisit as we don't want to rebuild the image too often.
+	local image_version=$(./release_notes.sh get-version)
```

Comment:
@Akos-Kreutz, usually, BChan doesn't like to add comments to the code unless it's strictly necessary, do you think this is the case for this comment?

Thanks

Reply:
Sure, I will remove the comment. I left it there because we might only use this feature years from now, so this knowledge might get lost, but it's not necessary to have it there.